### ISSUE TITLE
Missing Breeze animations

### ISSUE THREAD

**Issue by yeeters123e** [Mojang/bedrock-samples#47]
where are the breeze animations? I cant find it.

**Comment by Xterionix** [comment]
Non-issue. They won't be in the 1.20.XX files. It'll be in the 1.21 files

**Comment by MKokeshi** [comment]
The Breeze is a mob introduced in **Minecraft Legends**, but as of now, it hasn't been added to the standard Minecraft Bedrock Edition as an official entity with animations or behavior files. Therefore, you won't find animations for the Breeze mob in the **bedrock-samples** repository.

However, if you're working on a custom mod or addon for Minecraft Bedrock Edition and want to create animations for a custom mob like the Breeze, you would typically need to create these animations yourself using tools like **Blockbench** or by manually writing the necessary JSON files.

### Creating Custom Animations for a Mob

If you're interested in creating your own animations for a mob similar to the Breeze, here's a general guide on how to do it:

1. **Model the Mob:**
   - Use a tool like **Blockbench** to create a 3D model of the mob.
   - Define the different parts of the model (e.g., head, body, limbs) that you want to animate.

2. **Create Animations:**
   - In Blockbench, you can create animations for different actions (e.g., walking, attacking, idle).
   - Export these animations as JSON files compatible with Minecraft Bedrock Edition.

3. **Define Animations in the Behavior Pack:**
   - In your behavior pack, you would create or modify JSON files to include animation controllers that trigger these animations.
   - Here's an example of what an animation controller might look like:

   ```json
   {
       "format_version": "1.10",
       "minecraft:animation_controller": {
           "controller.animation.breeze.move": {
               "initial_state": "default",
               "states": {
                   "default": {
                       "animations": [
                           "move"
                       ],
                       "transitions": [
                           {
                               "walking": "query.is_moving"
                           }
                       ]
                   },
                   "walking": {
                       "animations": [
                           "breeze_walk"
                       ],
                       "transitions": [
                           {
                               "default": "!query.is_moving"
                           }
                       ]
                   }
               }
           }
       }
   }
   ```

   - This controller would trigger the `breeze_walk` animation when the mob is moving.

4. **Attach Animations to the Mob:**
   - In your resource pack, ensure that the animations are linked to the correct entity model. This is done in the `entity` JSON file where you define which animations to use for different actions.

### Example Structure

If you want to implement a custom Breeze with animations, your pack might have a structure like this:

```
behavior_pack/
|-- entities/
|   |-- breeze.json
|
resource_pack/
|-- animations/
|   |-- breeze_walk.animation.json
|-- models/
|   |-- breeze.geo.json
|-- textures/
|   |-- breeze.png
|-- entity/
|   |-- breeze.entity.json
```

### Summary

Since the Breeze from Minecraft Legends isn't a part of Bedrock Edition, you won't find official animations for it in the **bedrock-samples** repository. To add similar custom content, you'll need to create your own animations and integrate them into your behavior and resource packs using tools like Blockbench.
